src/services/videomania/index.jsx

```
@@ -248,3 +255,205 @@ export const AddVideoMania = ({
+        await getAllComments();
+      } else {
+        setComments(previousComments);
+      }
+    } catch (error) {
+      setComments(previousComments);
+      console.log(error);
+    }
+  };
+
+  const handleLike = async () => {
+    if (!video?.video_id) return;
```

ESLint

[error] 'video.video_id' is missing in props validation

(react/prop-types)